src/formatter/json_formatter.ts

```diff
@@ -9,6 +9,11 @@ import { getGherkinScenarioLocationMap } from './helpers/gherkin_document_parser
 import { ITestCaseAttempt } from './helpers/event_data_collector'
 import { doesHaveValue, doesNotHaveValue } from '../value_checker'
 import { parseStepArgument } from '../step_arguments'
+import ITag = messages.GherkinDocument.Feature.ITag
+import IFeature = messages.GherkinDocument.IFeature
+import IPickle = messages.IPickle
+import IScenario = messages.GherkinDocument.Feature.IScenario
+import IEnvelope = messages.IEnvelope
```

Comment:
Hmm, why are these import vs const?

Reply:
Yeah, I think you're right, I'll change and see if there's an eslint rule for that

Reply:
Okay this didn't actually work, so kept as it is. I'm trying to figure out why, maybe something relating to namespaces.